CRM/CivirulesActions/Participant/AddToZoom.php

```diff
@@ -31,13 +31,13 @@ public function processAction(CRM_Civirules_TriggerData_TriggerData $triggerData
 	 */
 	private function getWebinarID($event) {
 		$result;
-
+		$customField = CRM_NcnCiviZoom_Utils::getCustomField();
```

Comment:
In settings form, are we expecting the user to enter in the format 'custom_48'?.